src/hotspot/share/jvmci/jvmciCompilerToVM.cpp

```diff
@@ -1384,7 +1385,7 @@ C2V_END
 
 C2V_VMENTRY(void, invalidateHotSpotNmethod, (JNIEnv* env, jobject, jobject hs_nmethod, jboolean deoptimize))
   JVMCIObject nmethod_mirror = JVMCIENV->wrap(hs_nmethod);
-  JVMCIENV->invalidate_nmethod_mirror(nmethod_mirror, deoptimize, JVMCI_CHECK);
+  JVMCIENV->invalidate_nmethod_mirror(nmethod_mirror, deoptimize, nmethod::NMethodChangeReason::JVMCI_invalidate_nmethod, JVMCI_CHECK);
```

Comment:
Agree with @mhaessig's comment. If further discussion turns out to prefer the current `enum` over an `enum class`, then this `NMethodChangeReason::` is useless.